docs/getting_started/using_fixtures.md

```diff
@@ -20,6 +20,15 @@ Here's a top-level comparison of these two approaches:
 
     It's possible to execute `evm blocktest` directly within the execution-spec-tests framework. This is intended to verify fixture generation, see [Debugging `t8n` Tools](./debugging_t8n_tools.md).
 
+## How Test Fixtures are Consumed via Hive Simulators
+
+There are two Hive simulators that execute the test cases defined in the JSON test fixtures against fully instantiated client instances using the @ethereum/execution-apis:
+
+1. The `ethereum/pyspec` simulator executes the test cases against a fully instantiated execution client that receives blocks via the [Engine JSON-RPC API](https://github.com/ethereum/execution-apis/blob/main/src/engine/README.md). Each block is verified by the client and tested via the Engine API response. The post-state section of the test is additionally explicitly verified via the Engine API (post-Merge forks only).
```

Comment:
Nice! I relay what mario mentioned, I think we could add what rpc calls are used specifically.

Suggestion below :D

```suggestion
1. The `ethereum/pyspec` simulator consumes the test cases against a fully instantiated execution client that executes blocks via the [`engine_newPayloadVX`](https://github.com/ethereum/execution-apis/blob/main/src/engine/paris.md#engine_newpayloadv1) method. Each block is verified by the client and tested against the `engine_newPayloadVX` response. The post-state section of the test is additionally explicitly verified via `eth_getStorageAt` and `eth_getBalance` alongside an internal nonce checking method. Note the simulator only works for post-merge forks.
```

Reply:
Thanks a lot for this! 

Will just change `consumes` back to `executes` again, otherwise, thanks for improving this.